src/Sarif.Sarifer.Core/SpamBackgroundAnalyzer.cs

```diff
@@ -88,6 +88,7 @@ protected override bool AnalyzeCore(Uri uri, string text, string solutionDirecto
                 TargetUri = uri,
                 FileContents = text,
                 Logger = sarifLogger,
+                DynamicValidation = true, // Enable dynamic validations.
```

Comment:
>DynamicValidation = true [](http://example.com/codeflow?start=16&length=24)

wondering how long will the dynamic validations take to get the result. Maybe we can let user to decide if to enable the dynamic validation through an option?

Reply:
Valid point. Any latency changes are negligible at the moment. The greatest impact is going to be number of open tabs.
Letting the user decide isn't the best idea given the intent of this change. Ideally, we would want that to be an environment variable of some sort that requires administrator privileges to modify. This effort now falls outside of what is likely achievable to get to MVP for this week.